src/composables/fungibleTokens.ts

```diff
@@ -258,14 +258,25 @@ export function useFungibleTokens({ store }: IDefaultComposableOptions) {
     transaction: ITransaction,
     direction: string = TX_DIRECTION.sent,
   ) {
+    const isReceived = direction === TX_DIRECTION.received;
+
+    // This is out of place but since we are treating new protocols as fungible tokens
+    // it is better to have it here than in the protocol specific helper file
+    if (transaction.protocol !== PROTOCOL_AETERNITY) {
+      return new BigNumber(
+        transaction.tx?.amount || 0,
+      )
+        .plus(isReceived ? 0 : transaction.tx?.fee || 0)
+        .toNumber();
+    }
```

Comment:
Seriously we are treating BTC and ETH as fungible tokens? This is really bad. I haven't noticed this. Maybe we need to refactor this?